packages/esm-ward-app/src/ward-workspace/ward-patient-notes/history/delete-note.modal.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, ModalHeader, ModalBody, ModalFooter } from '@carbon/react';
+import { deleteEncounter } from '../../../ward.resource';
+import { showSnackbar } from '@openmrs/esm-framework';
+
+interface DeleteEncounterConfirmationProps {
+  encounterUuid: string;
+  close: () => void;
+  onDelete?: () => void;
+}
+
+const DeleteEncounterConfirmation: React.FC<DeleteEncounterConfirmationProps> = ({
+  close,
+  encounterUuid,
+  onDelete,
+}) => {
+  const { t } = useTranslation();
+  const handleCancel = () => close();
+
+  const handleDelete = async () => {
+    try {
+      await deleteEncounter(encounterUuid);
+      showSnackbar({
+        kind: 'success',
+        title: t('noteDeleteSuccessfully', 'Note deleted successfully'),
+      });
+      close();
+      onDelete();
+    } catch (e) {
+      showSnackbar({
+        kind: 'error',
+        title: t('errorDeletingNode', 'Error delete note'),
+        subtitle: e?.responseBody?.error?.translatedMessage ?? e?.responseBody?.error?.message,
+      });
+    }
+  };
+
+  return (
+    <>
+      <ModalHeader closeModal={close}>{t('deleteNote', 'Delete Note')}?</ModalHeader>
```

Comment:
I don't see the translations in this file in `en.json`. Did you mean to add them but forgot to commit them?

Reply:
Should we consider changing the script to something like `i18next 'src/**/*.{ts,tsx}' '!src/**/*.test.{ts,tsx}' ...`? I think that would basically do what we want in most cases...